### PR TITLE
Add dataset instruction

### PR DESCRIPTION
**Pull request by Louis-Dupont**
I made sure that we always have a link to where to download, as well as a tree representing the dataset structure.
Should we add something?

### CODE REVIEW

**Review by shaydeci — CHANGES_REQUESTED** [review]
Adding the structures in the dataset objects docs is nice to have but not what we aimed for.
We should add an additional section in the Readme.md "public datasets setup instructions" followed by the steps the users need to executer in order to start training with them in recipes.

File: src/super_gradients/training/datasets/Dataset_Setup_Instructions.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+            ├─val2012
+            ├─train2007
+            ├─test2007
+            └─val2007
+```
+</details>
+
+
+
+### Segmentation Datasets
```
Comment (shaydeci) [inline src/super_gradients/training/datasets/Dataset_Setup_Instructions.md:134]: Cityscapes ?

**Review by shaydeci — CHANGES_REQUESTED** [review]
Please add cityscapes instructions to Dataset_Setup_Instructions then LGTM.

File: src/super_gradients/training/datasets/Dataset_Setup_Instructions.md (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/training/datasets/Dataset_Setup_Instructions.md:134]: Added 709a1982fad825f5922bc20c72aa5b52b9b8fde1

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**